src/metadata/items.ts

```diff
@@ -33,7 +33,7 @@ export type Metadata = Details & {
   designer?: Designer;
 };
 
-export const EXTRA_DETAILS: Partial<Record<BumpkinItem, Details>> = {
+export const EXTRA_DETAILS: Required<Record<BumpkinItem, Details>> = {
```

Comment:
```suggestion
export const EXTRA_DETAILS: <Record<BumpkinItem, Details> = {
```

You shouldn't need to use `Required` here. Just by removing the `Partial` it will make every `BumpkinItem` be required.

Reply:
Ah I see, I'll made that change